Library/QuickPulseStateManager.ts

```diff
@@ -29,6 +29,8 @@ class QuickPulseStateManager {
     private _metrics: {[name: string]: Contracts.MetricQuickPulse} = {};
     private _documents: Contracts.DocumentQuickPulse[] = [];
     private _collectors: {enable: (enable: boolean) => void}[] = [];
+    private _rediectedHost: string = null;
+    private _poliingIntervalHint: number = -1;
```

Comment:
NIT: typos.  _redirectedHost and _pollingInterval

Reply:
Ah, thanks for pointing out.